application/helpers/expressions/em_core_helper.php

```diff
@@ -1190,70 +1190,110 @@ public function GetJavaScriptEquivalentOfExpression()
-                        if ($jsName != '') {
-                            $varName = $this->GetVarAttribute($token[0], 'varName', $token[0]);
-                            $stringParts[] = "LEMval('".$varName."') ";
+                switch ($token[2]) {
+                    case 'LP':
+                        $staticStringToParse .= $token[0];
+                        $bracket ++;
+                        break;
+                    case 'RP':
+                        $staticStringToParse .= $token[0];
+                        $bracket --;
+                        break;
+                    case 'DQ_STRING':
+                        // A string inside double quote : add double quote again
+                        $staticStringToParse .= '"'.$token[0].'"';
+                        break;
+                    case 'SQ_STRING':
+                        // A string inside single quote : add single quote again
+                        $staticStringToParse .= "'".$token[0]."'";
+                        break;
+                    default:
+                        // This set whole string inside function as a static var : must document clearly.
+                        $staticStringToParse .= $token[0];
+                }
+                if($bracket == 0) { // Last close bracket : get the static final function and reset
+                    //~ $staticString = LimeExpressionManager::ProcessStepString("{".$staticStringToParse."}",array(),3,true);
+                    $staticString = $this->sProcessStringContainingExpressions("{".$staticStringToParse."}",0,3,1,-1,-1,true); // As static : no gseq,qseq etc …
+                    $stringParts[] = $staticString;
+                    $staticFunction = null;
+                    $staticStringToParse = "";
+                }
+            } else {
+                switch ($token[2]) {
+                    case 'DQ_STRING':
+                        $stringParts[] = '"'.addcslashes($token[0], '\"').'"'; // htmlspecialchars($token[0],ENT_QUOTES,'UTF-8',false) . "'";
+                        break;
+                    case 'SQ_STRING':
+                        $stringParts[] = "'".addcslashes($token[0], "\'")."'"; // htmlspecialchars($token[0],ENT_QUOTES,'UTF-8',false) . "'";
+                        break;
+                    case 'SGQA':
+                    case 'WORD':
+                        if ($i + 1 < $numTokens && $tokens[$i + 1][2] == 'LP') {
+                            // then word is a function name
+                            $funcInfo = $this->RDP_ValidFunctions[$token[0]];
+                            if ($funcInfo[1] === null ) {
+                                /* start a static function */
+                                $staticStringToParse = $token[0]; // The function name
+                                $bracket = 0; // Reset bracket (again)
+                            } else {
+                                if ($funcInfo[1] == 'NA') {
```

Comment:
Usage of NA for JS function `myfunction("something")` => `"something"`

Maybe remove this checking ?